migrations/versions/407e74de5553_adding_user_table.py

```diff
@@ -8,14 +8,13 @@
 
 # revision identifiers, used by Alembic.
 revision = '407e74de5553'
-down_revision = '3a5eba38e4e8'
+down_revision = '12f9dcc2e36c'
```

Comment:
Kev and I have done some reordering to mend the history here - but it's still a bit dodgy and we need an answer so orderings are obvious.